### ISSUE TITLE
A typo was breaking builds

### ISSUE THREAD

**Issue by rosslibby** [apollographql/apollo-server#5214]
Today I used [patch-package](https://github.com/ds300/patch-package) to patch `apollo-server-core@2.24.0` for the project I'm working on.

I was getting the error "Argument of type '"uft8"' is not assignable to parameter of type 'BufferEncoding'."
The issue is that `utf8` was mistyped as `u**ft**8`.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/apollo-server-core/src/plugin/usageReporting/defaultSendOperationsAsTrace.ts b/node_modules/apollo-server-core/src/plugin/usageReporting/defaultSendOperationsAsTrace.ts
index 300a429..c003a33 100644
--- a/node_modules/apollo-server-core/src/plugin/usageReporting/defaultSendOperationsAsTrace.ts
+++ b/node_modules/apollo-server-core/src/plugin/usageReporting/defaultSendOperationsAsTrace.ts
@@ -23,7 +23,7 @@ export function defaultSendOperationsAsTrace() {
     // it.
     max: Math.pow(2, 20),
     length: (_val, key) => {
-      return (key && Buffer.byteLength(key, 'uft8')) || 0;
+      return (key && Buffer.byteLength(key, 'utf8')) || 0;
     },
   });
 
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by StarpTech** [comment]
Already published? https://github.com/apollographql/apollo-server/commit/bd44a4c6c4c69e9eb1f6f3514e8b8f6dfb14ce13

**Comment by glasser** [comment]
Hi! As @StarpTech noticed, this is already fixed on `main`, though I haven't put out a release containing it (I've been quite focused on the `release-3.0` branch) since I don't think it creates any *runtime* bugs. (As far as I understand, specifying an unknown encoding is the same as specifying `utf8`.)

(The issue wasn't detected previously because we hadn't upgraded `@types/node` in a while and the older version didn't make this into an error.)

My assumption has been that people doing builds against the published package are mostly going to be linking against the `.d.ts` files and so this wouldn't break them. I'll put out a patch release now off of main to resolve this. Thanks!

**Comment by glasser** [comment]
Published in 2.24.1.
